### PR TITLE
Added functionality to deploy PostgreSQL docker containers

### LINKED ISSUE #4: Add support to deploy Postgres SQL
https://hub.docker.com/_/postgres

Add support to deploy all releases of Postgres SQL for which the docker images are available.

### PR DESCRIPTION
**Pull request by silvs110**
closes #4
Currently only the latest PostgreSQL docker image is supported.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -78,6 +78,11 @@
             <artifactId>splunk</artifactId>
             <version>1.6.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>9.1-901-1.jdbc4</version>
```
Comment (the-c0d3br34k3r) [inline pom.xml:84]: Is there any reason why such an old version of the Postgres driver is used?
The latest version available in maven is 42.2.1.
https://mvnrepository.com/artifact/org.postgresql/postgresql

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
1st review complete. Have requested clarifications/changes.

File: pom.xml (hunk 1 above)
Comment (silvs110) [reply pom.xml:84]: My bad. I thought that was the latest.

Comment (the-c0d3br34k3r) [reply pom.xml:84]: Np

File: src/test/java/PostgreSQLDockerImageTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        logger.info("Executing query: " + query + " against PostgreSQL");
+        Statement statement = connection.createStatement();
+        ResultSet resultSet = statement.executeQuery(query);
+        int actualRowCount = 0;
+        while (resultSet.next()) {
+            logger.info("Query Result from PostgreSQL: " + resultSet.getString(1));
+            actualRowCount++;
+        }
+        // PostgreSQL have three databases created by default: postgres, template0, and template1
+       Assertions.assertEquals(3, actualRowCount);
```
Comment (the-c0d3br34k3r) [inline src/test/java/PostgreSQLDockerImageTest.java:41]: Spacing issue.

File: src/test/java/PostgreSQLDockerImageTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+            // Test if the connection to the database is successful
+            Assertions.assertDoesNotThrow(() -> connectToPostgreSQLDatabase(dockerImage));
+        }
+        finally {
+            // Stop and remove container
+            dockerUtility.removeContainerIfExists(containerName);
+        }
+    }
+
```
Comment (the-c0d3br34k3r) [inline src/test/java/PostgreSQLDockerImageTest.java:69]: Extra line can be removed.

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
2nd review complete.
Have requested more changes.
Please update the README.md file as well.

**Review by the-c0d3br34k3r — APPROVED** [review]

**Review by the-c0d3br34k3r — APPROVED** [review]

**Pull request merged**